hetzner_ddns.sh

```diff
@@ -13,7 +13,24 @@ for arg in $(seq "$#"); do
             printf '%s %s\n' "$self" "$version"
             exit 0;;
         '--help'|'-h')
-            man hetzner_ddns;
+            # man usually not available on OpenWrt (too big)
+            # use short help string in that case
+            if ! command -v man > /dev/null
+            then
+                printf '%s\n\n%s\n\n' "hetzner_dns 0.2.5 (2 August 2024) - Hetzner Dynamic DNS Daemon" "Usage hetzner_dns.sh [--daemon] [CONFIG]"
+                printf '%s\n' "This program runs as system service /etc/init.d/hetzner_ddns on OpenWrt."
```

Comment:
I would put every kind of configuration here like in the [man page](https://github.com/filiparag/hetzner_ddns/blob/0.2.5/hetzner_ddns.1.man#L24-L44), but you can leave that for after the PR.

Reply:
Sorry, I am not sure if I got you right: 
Should I describe service control with `service start|stop` etc in more detail similar to `systemctl` example in the man page? Would be glad to fix that as well, or if it is ok for you, I leave that for you to change?

Reply:
I was thinking about systemd-specifics like ["Multiple daemon instances"](https://github.com/filiparag/hetzner_ddns/blob/0.2.5/hetzner_ddns.1.man#L42-L44) section. It should either include all of them or none to keep it consistent.

You can leave it for later, as it doesn't affect the feature you have offered to implement 🙂